src/app/public/modules/timepicker/timepicker.component.ts

```diff
@@ -46,6 +50,9 @@ let nextId = 0;
 })
 export class SkyTimepickerComponent implements OnInit, OnDestroy {
 
+  /**
+   * Fires when the value in the timepicker input changes.
```

Comment:
This matches up with the description in the existing docs, but it is not surfacing in the docs in the PR preview. Is this another docs-tools issue?

Reply:
Yes: https://github.com/blackbaud/skyux-docs-tools/issues/68